package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -202,20 +165,9 @@ local function configure_mesh_wireless(radio, index, config)
 	local radio_name = radio['.name']
 	local suffix = radio_name:match('^radio(%d+)$')
 
-	local ibss_disabled = is_disabled('ibss_' .. radio_name)
-	local mesh_disabled = is_disabled('mesh_' .. radio_name)
-
-	configure_ibss(config.ibss(), radio, index, suffix,
-		first_non_nil(
-			ibss_disabled,
-			mesh_disabled,
-			config.ibss.disabled(false)
-		)
-	)
 	configure_mesh(config.mesh(), radio, index, suffix,
 		first_non_nil(
-			mesh_disabled,
-			ibss_disabled,
+			is_disabled('mesh_' .. radio_name),
```

Comment:
I think this is a corner case - But this way, the wireless enable-setting from IBSS is not transitioned to 11s, in case this is the first update to feature 11s. We need to save the enable-state for IBSS before deleting it's config.

Reply:
Users should migrate using scheduled-domain-switch on v2019.1.x, this just needs to be documented in the release notes.

Reply:
Okay, this is a valid point. :+1: